infrabox/test/api/collaborators_test.py

```diff
@@ -20,7 +20,7 @@ def test_collaborators_root(self):
         r = TestClient.post('api/v1/projects/%s/collaborators' % self.project_id,
                             data=self.test_collaborator_data,
                             headers=TestClient.get_user_authorization(self.user_id))
-        self.assertEqual(r['message'], 'Successfully added user')
+        self.assertEqual(r['message'], 'Successfully added user.')
```

Comment:
no . at the end

Reply:
I added this to keep consistency across the code. Since some requests return with dot at the end and some without. We can consider to remove dots at the end everywhere, but always add a dot in the logger.

What do you think?